src/runner.ts

```diff
@@ -121,8 +121,8 @@ export async function run(options: Options, logger: Logger): Promise<Status> {
     try {
         return await runWorker(options, logger);
     } catch (error) {
-        if ((error as FatalError).name === FatalError.NAME) {
-            logger.error((error as FatalError).message);
+        if (error instanceof FatalError) {
```

Comment:
I would generally avoid `instanceof`, but in this case it's fine since `FatalError` is declared within the tslint package. If it was an import from another package, `instanceof` is prone to fail depending on which version of that package is used at runtime (there could be multiple versions installed in `node_modules`).